components/01-atoms/images/icons/_icon.twig

```diff
@@ -11,7 +11,16 @@
  */
 #}
 {% set icon_base_class = icon_base_class|default('icon') %}
-{% set icon_url = icon_url|default('icons.svg#') %}
+{% if directory %}
+  {% set icon_url = icon_url|default('/' ~ directory ~ '/dist/') %}
+{% else %}
+  {% set icon_url = '' %}
```

Comment:
Since the variable below doesn't render anything if it isn't passed anything, we can just delete lines 16-17, and it'll still work as expected

Reply:
Also, I don't think we need the `icon_url|default` stuff? That was just to allow the global data file to override it, but we're using the if/else to do that now.

And what do you think about using a ternary?
`{% set icon_url = directory ? '/' ~ directory ~ '/dist/' %}`

Reply:
Done